modules/history/src/views/full/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import style from './style.scss'
+import moment from 'moment'
+
+import { MessagesViewer } from './MessageViewer'
+import { ConversationPicker } from './ConversationPicker'
+
+const CONV_PARAM_NAME = 'conversation'
+
+export default class FullView extends React.Component {
+  state = {
+    conversationsInfo: [],
+    messageGroups: [],
+    to: moment().startOf('day'),
+    from: moment()
+      .startOf('day')
+      .subtract(30, 'days'),
+    currentConversation: null
+  }
+
+  componentDidMount() {
+    this.getConversations(this.state.from, this.state.to)
+    const url = new URL(window.location.href)
+    const sessionId = url.searchParams.get(CONV_PARAM_NAME)
+    if (sessionId) {
+      this.setState({ currentConversation: sessionId })
```

Comment:
getMessagesOfConversation already sets the `sessionId`.  I'd either let this method set it, or at least set `messageGroups` and `conversationsInfo` to `null` while it's loading

Reply:
:+1: